frontend/src/components/PostsList.js

```diff
@@ -1,11 +1,22 @@
 import React from 'react'
 import Post from './Post'
 import '../App.css'
+import { useParams } from 'react-router-dom'
+import { useDispatch, useSelector } from 'react-redux'
+import { saveCategoryPosts } from '../store/actions/actions'
 
 const PostsList = props => {
   const { posts } = props
+  const categories = useSelector(state => state.categories)
+  const { id } = useParams()
+  const dispatch = useDispatch()
 
   if (!posts) {
+    const findCategory = categories.find(({ name }) => name === id)
```

Comment:
`findCategory` sound more like a function. I suggest `foundCategory` or just `category`

Reply:
yea i have to improve naming, but `foundCategory` sound nice, in this case when find one